components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImpl.java

```diff
@@ -64,6 +63,17 @@ public void httpGet(String endpointURL, Object... params) {
                             "and eventHandlers (Map<String, Object>) respectively.");
                 }
                 break;
+            case 3:
+                if (params[0] instanceof Map && params[1] instanceof Map && params[2] instanceof Map) {
+                    authConfig = (AuthConfigModel) params[0];
+                    headers = validateHeaders((Map<String, ?>) params[1]);
+                    eventHandlers = (Map<String, Object>) params[2];
```

Comment:
This is not the order we agreed in the proposal to process the function right.
That order is: uri, headers, authconfig, callback handlers

Reply:
Fixed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/157/commits/dc7ea3e7070ae1a51e26d6d777bb50b4e410581b